common/build/build-common/.changeset/rich-candles-tan.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
@tylerbutler Curious about the file names. I'm assuming they're autogenerated, and we don't expect devs to go looking through them?

Reply:
Correct, auto-generated but meaningless. We can rename them, or use our own file names, but the key is that they're meaningless and the randomness should prevent merge conflicts.